src/components/ImportHistoryPage.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <div class="text-center min-h-screen">
+        <h1 class="text-center my-2">Import History</h1>
+        <form>
+            <br />
+            <div>
+                <input ref="fileSelector" type="file" @change="fileChange" />
```

Comment:
We should style that file selector similar to the one from the playlists page, see https://github.com/TeamPiped/Piped/blob/master/src/components/PlaylistsPage.vue#L13-L14.